pkg/enemy/enemy.go

```diff
@@ -19,6 +19,12 @@ type EnemyProfile struct {
 	ParticleDropThreshold float64                        `json:"-"` // drop particle every x dmg dealt
 	ParticleDropCount     float64                        `json:"-"`
 	ParticleElement       attributes.Element             `json:"-"`
+	ParticleDrops         []HpDrop                       `json:"-"`
+	ResistFrozen          bool                           `json:"-"`
+	HpBase                float64                        `json:"-"`
+	HpGrowCurve           int                            `json:"-"`
+	Id                    int                            `json:"-"`
+	MonsterName           string                         `json:"-"`
```

Comment:
Nice to have: a `modified` bool which will tell me if this `EnemyProfile` deviated from the default. Then in UI I can prefix that enemy as modified/custom to make it clear it is not default

Reply:
Yeah so my thought is either we expose the original `EnemyProfile` in the stats as well and show a diff, or have a `modified` bool and we just indicate that "something has deviated from the original profile" but leave it to the viewer to figure out what